Funcky/Monads/Reader/Reader.Transformation.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Funcky.Monads
+{
+    public static partial class ReaderExtensions
+    {
+        public static Reader<TEnvironment, IEnumerable<TElement>> Sequence<TEnvironment, TElement>(this IEnumerable<Reader<TEnvironment, TElement>> sequence)
```

Comment:
Is this related to #344?

Reply:
Yes, that is one of those functions and the one I need currently, the name is borrowed from lang-ext.